charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -3,19 +3,30 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 
-alfresco-process-services:
+
+alfresco-infrastructure:
   enabled: true
-  alfresco-infrastructure:
-    enabled: false
-  processEngine:
-    environment:
-      IDENTITY_SERVICE_ENABLED: true
-      IDENTITY_SERVICE_AUTH: "<ingresscontrollerurl>/auth"
-  resources:
-    requests:
-      memory: "2000Mi"
-    limits:
-      memory: "3000Mi"
+  alfresco-identity-service:
+    realm:
+      alfresco:
+        client:
+          redirectUris:
+            - "*"
+          webOrigins:
+            - "*"
+  persistence:
+    efs:
+      enabled: true
+      dns: "$NFSSERVER"
```

Comment:
Are we starting to assume AWS as the default deployment rather than providing AWS-specific instructions in the README?

Won't these changes break the Docker for Desktop instructions without modification?

Reply:
Do we still want to deploy this on Docker for Desktop?

Reply:
That seems to continue to be a business requirement.  We may be able to make further tweaks to trim it down.

Reply:
We now have minimal-values.yaml